CHANGES.md

```diff
@@ -0,0 +1,21 @@
+## v0.1.1
+
+Conduit 0.1.1 is focused on making it easier to get started with Conduit.
+
+* Conduit can now be installed on Kubernetes clusters that use RBAC
+* The new `--skip-outbound-ports` flag for the CLI `inject` command directs Conduit to bypass proxying for specific outbound ports, making Conduit easier to use with non-gRPC or HTTP/2 protocols.
+* The CLI `tap` command output has been reformatted to be line-oriented, making it easier to parse with common UNIX command line utilities
+* Proxies now support routing to non-fully qualified domain names, with routing rules that more closely resemble the default Kubernetes DNS search path
```

Comment:
Proxies are an impl detail -- the user facing change is that Conduit can now route requests that use DNS names that are relative to the default kubernetes search path...

> Conduit now supports routing of non-fully qualified domain names.

(I'm not sure the routing rule info is user-facing, really)

Reply:
Heh, that's how it was originally written. Will switch it back.